include/swift/Basic/TaskQueue.h

```diff
@@ -39,6 +41,57 @@ enum class TaskFinishedResponse {
   StopExecution,
 };
 
+/// TaskProcessInformation contains information about the process a task was
+/// associated with.
+struct TaskProcessInformation {
```

Comment:
I'm still looking for more info concerning how this information will or might be used.

Reply:
Also `TaskProcessInformation` is OK, if a bit awkward. "Information" could be anything, including the execution arguments. Is there a more specific term? Right now, its "times". Are you planning to generalize? Or would "TaskExecutionStatistics" cover it?

Reply:
Now that this contains the real_pid I see it as a container for all 'information' about the OS process that ran. The times/usage are one part of it, real_pid is the other (for now). I don't plan to add more to this but it could contain other information like the user who started it (which is pretty useless in this case).